### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by DeryabinSergey**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | -

### CODE REVIEW

File: src/clients/Yandex.php [hunk 1]
```diff
@@ -62,6 +62,24 @@ protected function initUserAttributes()
         return $this->api('info', 'GET');
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getUserAttributes()
```
Comment (samdark) [inline src/clients/Yandex.php:68]: I don't think we should do normalization at this level. It isn't done for any other client and client isn't expected to always return `email`.

Comment (DeryabinSergey) [reply src/clients/Yandex.php:68]: In doc [example](https://www.yiiframework.com/extension/yiisoft/yii2-authclient/doc/guide/2.0/ru/quick-start#dobavlenie-eksena-v-kontroller) waiting only email and nothing else )

Comment (samdark) [reply src/clients/Yandex.php:68]: Well, in the docs example it's described how to achieve it: https://github.com/yiisoft/yii2-authclient/blob/d34ee5d924bfb151d6b4028c57c57dd4b18aedf9/src/BaseClient.php#L69

Comment (DeryabinSergey) [reply src/clients/Yandex.php]: Sorry for loooong contribute, facepalm

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Thanks!

**Pull request merged**